### DISCUSSION TITLE
Welcome to quill_html_editor Discussions!

### DISCUSSION THREAD

**Discussion by the-airbender** [the-airbender/quill_html_editor#32] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by biYTccKCiu** [comment]
Can't see anything about custom attributes for metatdata here. Someone please correct me if I'm wrong. What I am looking to do , is to add attribute tags to text such as : ` <PersonRef key='SAdam'>Stuart</PersonRef>   `where the tag PersonRef would colour the text... 

is this at all possible?

**Comment by AlicyberMeteors** [comment]
When route to previous screen editor sometimes jitter on screen and screen flicks. How can i fix
